.gitlab-ci.yml

```diff
@@ -10,15 +10,15 @@ build:
     - "export DOCKER_HOST=tcp://localhost:2375"
     - "docker network prune -f && docker network create amazeeio-network"
     - "docker login -u gitlab-ci-token -p $CI_JOB_TOKEN gitlab-registry-production.govcms.amazee.io"
-    - "ahoy up"
+    - "docker-compose up -d"
     - "docker-compose exec -T test dockerize -wait tcp://mariadb:3306 -timeout 1m"
     - |
-      if grep --quiet '^MARIADB_DATA_IMAGE' .env; then
-        ahoy govcms-deploy
-      else
-        ahoy -v composer install
-        ahoy -v install
+      if ! grep --quiet '^MARIADB_DATA_IMAGE' .env; then
+        echo "MARIADB_DATA_IMAGE is not set, installing the govcms profile."
+        docker-compose exec -T cli composer install
+        docker-compose exec -T cli drush si -y govcms
       fi
+      docker-compose exec -T cli govcms-deploy
```

Comment:
I've replaced some ahoy lines here, and made it a bit cleaner while I was debugging. The old way was running "ahoy govcms-deploy" inside ahoy -v install, which was very not clear. I'd argue that ahoy -v install shouldn't do a deploy based on the semantics of the name. Either way the new way is much more transparent imo. I also added a clear message when MARIADB_DATA_IMAGE is not set - as the implications are that the project is not completely initialised and there is still something to do.

Reply:
do we need to do the composer install twice?

Reply:
We don't! i was meant to move it but duplicated it. Fixed now.